packages/react-components/react-field/src/components/Field/useField.tsx

```diff
@@ -118,20 +118,14 @@ export const useField_unstable = <T extends FieldComponent>(
     control['aria-labelledby'] ??= label.id;
   }
 
+  if (validationMessage || hint) {
+    // The control is described by the validation message, or hint, or both
+    control['aria-describedby'] ??=
+      validationMessage && hint ? `${validationMessage.id} ${hint.id}` : validationMessage?.id || hint?.id;
```

Comment:
Would we want to prepend the validation message id to an existing `aria-describedby` if one is present?

I understand not wanting to append or overwrite in the case of `hint`, since a hint is probably equivalent in purpose to most use cases of author-provided `aria-describedby`. Error messages seem different, since the most common use case would be using the error in addition to the provided descriptive text. I actually can't easily think of a case where one wouldn't want that to happen.

If we don't add the validation id to an existing attribute value, is there an easy way for an author provide both a custom `aria-describedby` and a validation message?

Reply:
We've generally used the approach that if someone explicitly sets a prop like `aria-describedby="foo"`, we let that override anything that the component would set internally. Otherwise, there's no way for them to specify the value if they truly want to replace what the component sets.

If someone wanted to append something else to a field's aria-describedby, they can set their own ID on the validationMessage, and reference that ID in aria-describedby.

```jsx
<InputField
  validationMessage={{ id: 'my-validation-message', children: '...' }}
  aria-describedby="my-validation-message foo"
/>
```

That is a bit complicated. I don't really expect people to need to do it, but at least it's possible.

Reply:
Talked to @smhigley offline, and we decided to make Field append any user-supplied `aria-describedby` to the validationMessage and hint, rather than override it:
1. Users may have custom descriptions outside the field, but we don't want that to unexpectedly override the error message.
2. If someone adds a Tooltip to a field, then that could set `aria-describedby`, but it shouldn't override the error message.
3. The failure condition is safer: in the case where the user supplies a custom `aria-descibedby`, then the worst case is that there is some redundant information in the description.
4. It is still technically possible to completely override via `<InputField control={{ 'aria-describedby': 'my-description' }} />`